x: -0.9
y: 1.07
th: 0.00
cov_x: 0.00
cov_y: 0.00
cov_th: 0.00
